overlays/DIMO/dimo.vspec

```diff
@@ -71,3 +71,15 @@ Vehicle.IsIgnitionOn:
   type: sensor
   datatype: boolean
   description: Vehicle ignition status. False - off, True - on.
+
+Vehicle.Powertrain.TractionBattery.Charging.ChargeCurrent.AC:
+  datatype: float
+  type: sensor
+  unit: A
+  description: Current AC charging current (rms) at inlet. Negative if returning energy to grid. Used when per-phase numbers are unavailable.
```

Comment:
maybe spell out what rms means

Reply:
The thing is they use it without comment in the main file.